VargasMartinezJose/Dia3/MiniProject/NewPlayer.java

```diff
@@ -0,0 +1,108 @@
+package miniproject.bootcamp.np;
+
+public class NewPlayer {
```

Comment:
Change this class name as "Player", the word New is redundant, when yo create a new instans, you created a new Player